src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -12,8 +12,9 @@
 <c:set var="FEEDBACK_STUDENTS"><%=FeedbackParticipantType.STUDENTS.name()%></c:set>
 <c:set var="FEEDBACK_INSTRUCTORS"><%=FeedbackParticipantType.INSTRUCTORS.name()%></c:set>
 
-<div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
+<div class="col-sm-12 margin-bottom-15px padding-15px <%= fqForm.isQuestionHasResponses() ? "alert alert-danger" : "background-color-light-green" %>">
     <div class="margin-bottom-7px">
+        <c:if test="${fqForm.questionHasResponses}"><h4>Changing the visibility after collecting responses is not recommended.</h4></c:if>
```

Comment:
I guess no need for italics. May be add some space after the 2nd line so that it appears separated from the `Visibility` line? Ideally, the two sentences should run together rather than appear in two lines but I guess that's hard to do if you use `<h4>`.